### ISSUE TITLE
can not get  videos.streamsClient.getHttpLiveStreamUrl

### ISSUE THREAD

**Issue by Joseph-Nathan** [Hexer10/youtube_explode_dart#134]
**Describe the bug**
i can not use getHttpLiveStreamUrl to get liveURL

**To Reproduce**
```dart
      streamURL = await retry(() => yt.videos.streamsClient.getHttpLiveStreamUrl(VideoId("https://www.youtube.com/watch?v=2PeFhZg-Ci4")) , 20) ;

```

**Stacktrace**
Include here the stack trace (if applicable). 

**Enviroment: (please complete the following information):**
 - Enviroment: [Flutter ]
 - Version [Channel stable, 2.2.0, on Microsoft Windows [Version 10.0.19042.1052], locale en-US]
 - YoutubeExplode Version [e.g. ^1.9.4]

**Comment by Hexer10** [comment]
Unfortunately seems like youtube doesn't send that url anymore with the `video player info`. It appears in the WatchPage initial data, so we might get it from there.
See https://github.com/Tyrrrz/YoutubeExplode/issues/566

**Comment by Joseph-Nathan** [comment]
in exp to how scrap it or tag id or key ??

**Comment by Hexer10** [comment]
I just published a new version (v`1.9.9`) which should address this isssue! Let me know if you still have some issues.
